src/lib.rs

```diff
@@ -53,7 +53,8 @@ mod client;
 pub use client::Client;
 mod endpoints;
 pub use endpoints::{
-    accounts::{Account, Owner, Type as AccountType},
+    accounts,
+    accounts::{Account, Owner},
```

Comment:
can you add a `#[doc(inline)]` attribute here so that `Account` and `Owner` only appear in the docs in one place?

Reply:
Is this what you meant? I'm not familiar with `#[doc(inline)]`..